micromeow/fosterfinder
# Feat: Add animal card and render them in app page

# Description

This PR adds `AnimalCard` to display registered animals with status.

## Changes

- Add `AnimalCard` with tests and stories
- Add `getAnimals` with tests
- Add animal cards to `/app` page
- Add animals and admin user to seed

## Testing
1. `npm i`
2. `npm run dev`
3. `npm run db:reset`
4. Login and navigate to `/app`
5. Animal cards should be visible
6. `npm run storybook`
7. Click on `AnimalCard` docs
8. Storybook should correctly display `AnimalCard`
9. `npm test`
10. All tests should pass

## Screenshots
<img width="1727" alt="image" src="https://github.com/micromeow/fosterfinder/assets/13270108/79e5f5c5-feaf-4243-8fe2-5f86da77d613">
